script.js

```diff
@@ -1,12 +1,46 @@
 //You can edit ALL of the code here
+const getBody = document.querySelector("body");
```

Comment:
`getBody` sounds like a function and not the name of the body element here - watch out